.github/workflows/main.yml

```diff
@@ -23,10 +23,7 @@ jobs:
       - name: yarn install
         run: yarn install --frozen-lockfile
 
-      - name: Test
-        run: yarn test --ci --coverage
-
       - name: Publish
```

Comment:
Should we not test before publishing? :)

Reply:
Well. I'm not proud of this, but at the moment there aren't any tests to run.

Reply:
Aah okay.